ui/src/main/java/com/epam/eco/kafkamanager/ui/topics/browser/ToTabularRecordsConverter.java

```diff
@@ -63,7 +64,16 @@ private static Record toTabularRecord(
                 valueTabulator.getAttributes((ConsumerRecord)consumerRecord);
         Map<String, String> headers = new HashMap<>();
         consumerRecord.headers().forEach( header -> headers.put(header.key(),new String(header.value())));
-        return new Record(consumerRecord, tabularValue, attributes, headers);
+        AvroSchema avroSchema = null;
+        if(valueTabulator instanceof AvroRecordValueTabulator) {
+            AvroRecordValueTabulator avroValueTabulator = (AvroRecordValueTabulator)valueTabulator;
+            ConsumerRecord<?, Object> avroConsumerRecord = (ConsumerRecord<?, Object>)consumerRecord;
+            Schema schema = avroValueTabulator.getSchema(avroConsumerRecord);
+            avroSchema = new AvroSchema(avroConsumerRecord.topic()+"-key", avroConsumerRecord.topic()+"-value",
```

Comment:
Please avoid using schema key and value as we can't know them for sure, topic+"-key" and topic+"-value" is just one of many naming strategies available in schema registry